src/smart-components/ChannelList/components/ChannelPreview/index.tsx

```diff
@@ -27,6 +27,7 @@ interface ChannelPreviewInterface {
   channel: GroupChannel;
   isActive?: boolean;
   isTyping?: boolean;
+  isEphmeral?: boolean;
```

Comment:
dont need to drill down this prop, channel is already present as a prop, no?

Reply:
Ah, it's a good catch, and why there's also a typo 😞🙀 my bad..